### DISCUSSION TITLE
New versioning scheme

### DISCUSSION THREAD

**Discussion by p91paul** [dbeaver/dbeaver#11024] [DBeaver Development] [answered]
Hi, noticed in the milestones section that after v7 you'll switch to v21 and I got curious. I assume you'll dump major version numbers yearly from now on? What will happen for minor releases, will they retain the current cadence and numbering?

**Comment by serge-rider** [comment, marked as answer]
Yes, we decided to switch to "calendar" versioning. 
Rules for minor releases remain the same (second number changes quarterly, third number changes biweekly).
